### PR TITLE
LADX: Add --no-magpie argument for disabling magpie bridge

### PR DESCRIPTION
**Pull request by Cybrou**
## What is this fixing or adding?
It adds an command line argument --no-magpie to LinksAwakeningClient which allows to disable the embed magpie bridge.
It allows to use an other autotracking method for magpie or simply to prevent the connection attempts to the port 17026 if you have something else that use it.

## How was this tested?
I played a partial seed with and without, I got the expected result (no magpie connection with the argument, functionnal autotracking without it) 

## If this makes graphical changes, please attach screenshots.

Without the --no-magpie switch:
image

With the --no-magpie switch:
image

### CODE REVIEW

**Comment by Berserker66** [comment]
@zig-for

File: LinksAwakeningClient.py [hunk 1]
```diff
@@ -570,9 +579,11 @@ async def deathlink():
 async def main():
     parser = get_base_parser(description="Link's Awakening Client.")
     parser.add_argument("--url", help="Archipelago connection url")
+    parser.add_argument("--nomagpie", default=False, action='store_true', help="Disable magpie bridge")
```
Comment (zig-for) [inline LinksAwakeningClient.py:582]: `--no-magpie` please

Comment (zig-for) [reply LinksAwakeningClient.py:582]: Even better
 `   parser.add_argument("--no-magpie", dest='magpie", default=False, action='store_false`, help="Disable magpie bridge")`
then pass `args.magpie` to the context.

**Review by zig-for — CHANGES_REQUESTED** [review]
LGTM other than swapping the bool (bool variables should almost never start with "no" - just invert them.

**Comment by Cybrou** [comment]
Done ! That dest argument is handy.

**Review by Berserker66 — APPROVED** [review]

**Pull request merged**